std/algorithm/sorting.d

```diff
@@ -19,6 +19,8 @@ $(T2 isPartitioned,
         afterwards.)
 $(T2 isSorted,
         $(D isSorted([1, 1, 2, 3])) returns $(D true).)
+$(T2 isSortedStrictly,
```

Comment:
I can't find any justification for it. The link you posted seems to concern `REF` (sic, use `LREF`) vs `D`